drive/metadisplayer/metadisplayer.html

```diff
@@ -0,0 +1,164 @@
+      var API_KEY = "<YOUR_API_KEY>";
+
+      // The Client ID obtained from the Google API Console. Replace with your own Client ID.
+      var CLIENT_ID = "<YOUR_CLIENT_ID>";
+
+      // The APP ID obtained from the Google API Console. Replace with your own APP ID and
+      // keep in sync with API_KEY and CLIENT_ID.
+      var APP_ID = "<YOUR_APP_ID>";
+
+      // API discovery doc URL for APIs used by this example
+      var DISCOVERY_DOCS = "https://www.googleapis.com/discovery/v1/apis/drive/v3/rest";
+
+      // Authorization scopes required by the API; multiple scopes can be
+      // included, separated by spaces.
+      var SCOPES = 'https://www.googleapis.com/auth/drive.file';
+
+      var GoogleAuth;
+      var GoogleUser;
+      var pickFileButton = document.getElementById('pick_file_button');
+
+      pickFileButton.onclick = handlePickFileButtonClick;
+
+      /**
+       *  On load, called to load the API client library, the auth2 library, and the picker library.
+       */
+      function handleClientLoad() {
+        gapi.load('client:auth2:picker', initClient);
+      }
+
+      /**
+       *  Initializes the API client library and sets the authorization and current user.
+       */
+      function initClient() {
+        gapi.client.init({
+          client_id: CLIENT_ID,
+          discoveryDocs: [DISCOVERY_DOCS],
+          scope: SCOPES
+        }).then(function () {
+          GoogleAuth = gapi.auth2.getAuthInstance();
+          GoogleUser = GoogleAuth.currentUser.get();
+          pickFileButton.style.display = 'block';
+        }, function(error) {
+          appendPre(JSON.stringify(error, null, 2));
+        });
+      }
+
+      /**
+       * Ensures the user has the correct scopes prior to displaying the picker.
+       */
+      function handlePickFileButtonClick() {
+        if (!GoogleUser.hasGrantedScopes(SCOPES)) {
+          GoogleUser.grant({
+            scope: SCOPES
+          }).then(displayPicker, function(err) {
+            // Handle error/declined auth...
+          });
+        } else {
+          displayPicker();
+        }
+      }
+
+      /**
+       * Create and render a Picker for selecting any file on My Drive.
+       */
+      function displayPicker() {
+        // Fetch current token
+        let authResponse = GoogleUser.getAuthResponse(true);
+
+        var view = new google.picker.View(google.picker.ViewId.DOCS);
+        var picker = new google.picker.PickerBuilder()
+          .setAppId(APP_ID)
+          .setOAuthToken(authResponse.access_token)
+          .addView(view)
+          .setDeveloperKey(API_KEY)
+          .setCallback(pickerCallback)
+          .build();
+        picker.setVisible(true);
+      }
+
+      /**
+       * Called when a file is picked. This function gathers metadata from
+       * the picked documents and from calling get on the fileId.
+       */
+      function pickerCallback(data) {
+        var document = data[google.picker.Response.DOCUMENTS][0];
```

Comment:
Ideally should also handle case where user cancels and selects nothing. Can defer that to a follow up PR if needed.